soapboxd/application.go

```diff
@@ -385,6 +417,80 @@ func setPbTimestamp(ts *gpb.Timestamp, t time.Time) {
 	ts.Seconds = t.Unix()
 }
 
+func (s *server) DeleteApplication(ctx context.Context, app *pb.Application) (*pb.Empty, error) {
+	go s.deleteAppInfrastructure(app)
```

Comment:
Seems like we'd want to propagate the ctx down to the go routine so it's cancelled if the parent context gets cancelled. But i haven't traced this up to see where that context originates (or if there's even cancelling logic there).

So more a food for thought comment than a must do.